hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -653,8 +707,12 @@ private void logVersionMismatch(OzoneConfiguration conf, ScmInfo scmInfo) {
   private void instantiateServices(boolean withNewSnapshot) throws IOException {
 
     metadataManager = new OmMetadataManagerImpl(configuration);
-    multiTenantManager = new OMMultiTenantManagerImpl(this, configuration);
-    OzoneAclUtils.setOMMultiTenantManager(multiTenantManager);
+    LOG.info("S3 Multi-Tenancy is {}",
+        isS3MultiTenancyEnabled ? "enabled" : "disabled");
```

Comment:
Left over debug statement? We should have covered necessary logging for enabling/disabling in the OM constructor.

Reply:
Nope. If multi-tenancy is successfully enabled there will be no message printed earlier in the validation phase at all. This INFO log is put here deliberately (and just in case we support changing config keys during reload).